### PR TITLE
[6669] apps/exports: change wording for the moderator feedback fields

### PR DESCRIPTION
**Pull request by fuzzylogic2000**
Well, we kind of switched the names now, so the statement is called feedback and what we call feedback in the code is the status now. :cry: I guess, we also need to at least adapt the verbose names on a+ or it will be veeeery confusing. But well, for the current mB story, this is it.

### CODE REVIEW

**Comment by Rineee** [comment]
Uh yes, confusing.. but we did confirm this wording change with Aplus, right? @CarolingerSeilchenspringer

**Comment by fuzzylogic2000** [comment]
@khamui is currently doing the renaming. He can also do it in here when the models are through in mB and a+.

File: tests/apps/ideas/models.py [hunk 1]
```diff
@@ -21,9 +21,9 @@ class Idea(Item):
     name = models.CharField(max_length=120,
                             default='Can i haz cheezburger, pls?')
     description = RichTextField(verbose_name='Description', blank=True)
-    moderator_feedback = models.CharField(max_length=256, blank=True)
-    moderator_statement = models.OneToOneField(
-        ModeratorStatement,
+    moderator_status = models.CharField(max_length=256, blank=True)
+    moderator_feedback_text = models.OneToOneField(
```
Comment (Rineee) [inline tests/apps/ideas/models.py:25]: To me `moderator_feedback` makes more sense here? And then the text is `moderator_feedback.feedback_text` .. why did you name it `moderator_feedback_text`?

**Review by Rineee — APPROVED** [review]
Yes, nice!

**Pull request merged**